wsclient/src/main/java/org/hpccsystems/ws/client/platform/DropZone.java

```diff
@@ -49,6 +50,35 @@ public String getName()
         return info.getName();
     }
 
+    public String[] getConfNetAddresses()
+    {
+    	if (info.getTpMachines() != null && info.getTpMachines().length > 0)
+    	{
+    		String[] cnaList = new String[info.getTpMachines().length];
+    		for(int i = 0; i < info.getTpMachines().length; i++)
+    		{
+    			cnaList[i] = new String(info.getTpMachines()[i].getConfigNetaddress());
+    		}
+    		return cnaList;
+    	}
+    	return null;
+    }
+    
+    public String[] getNetAddresses()
+    {
+    	if (info.getTpMachines() != null && info.getTpMachines().length > 0)
+    	{
+    		String[] naList = new String[info.getTpMachines().length];
+    		for(int i = 0; i < info.getTpMachines().length; i++)
+    		{
+    			naList[i] = new String(info.getTpMachines()[i].getNetaddress());
+    		}
+    		return naList;
+    	}
+    	return null;
+    }
+    
+    // Deprecated
```

Comment:
Use the Java notation:
/**
 * 
 * @deprecated Do not use this method!
 */
 @Deprecated